docs/quickstart.md

```diff
@@ -49,7 +49,7 @@ See [https://docs.microsoft.com/en-us/azure/developer/terraform/get-started-clou
 
 ### Initialise Terraform and deploy
 
-To initalise terraform, from the root of the repository run:
+To initialise terraform, from the root of the repository run:
```

Comment:
Nit: We should be consistent with UK vs US English. Let's decide and update docs accordingly, right now it's a bit of a mix.